code/modules/mob/inventory.dm

```diff
@@ -219,87 +219,92 @@
 
 /mob/living/carbon/human/proc/equip_if_possible(obj/item/W, slot, del_on_fail = 1) // since byond doesn't seem to have pointers, this seems like the best way to do this :/
 	//warning: icky code
-	var/equipped = 0
+	var/equipped = FALSE
+	var/in_storage = FALSE
 	switch(slot)
 		if(SLOT_BACK)
 			if(!src.back)
 				src.back = W
-				equipped = 1
+				equipped = TRUE
 		if(SLOT_WEAR_MASK)
 			if(!src.wear_mask)
 				src.wear_mask = W
-				equipped = 1
+				equipped = TRUE
 		if(SLOT_HANDCUFFED)
 			if(!src.handcuffed)
 				update_handcuffed(W)
-				equipped = 1
+				equipped = TRUE
 		if(SLOT_L_HAND)
 			if(!src.l_hand)
 				src.l_hand = W
-				equipped = 1
+				equipped = TRUE
 		if(SLOT_R_HAND)
 			if(!src.r_hand)
 				src.r_hand = W
-				equipped = 1
+				equipped = TRUE
```

Comment:
I'm fixing this using flags, but this will cause problems, because holding stuff on the hands will trigger as equipped.

Reply:
Should hands also be considered `in_storage`? Or are we planning on having hand held shields in the future?

Reply:
> 
> 
> I'm fixing this using flags, but this will cause problems, because holding stuff on the hands will trigger as equipped.

literally see #3391